dashboard/lib/contact_rollups_v2.rb

```diff
@@ -1,52 +1,85 @@
+require 'cdo/log_collector'
+
 class ContactRollupsV2
-  def self.build_contact_rollups(log_collector, sync_with_pardot=false)
-    log_collector.time!('Deletes intermediate content from previous runs') do
+  def initialize
+    @log_collector = LogCollector.new
+  end
+
+  # Build contact rollups and sync the results to Pardot.
+  def build_and_sync
+    collect_and_process_contacts
+
+    # These sync steps are independent, one could fail without affecting another.
+    # However, if the build step above fails, none of them should run.
+    sync_new_contacts_with_pardot
```

Comment:
So we have no more `sync_to_pardot` flag, Pardot steps will happen by default. Is that a problem?

Reply:
Right, I forgot about that! In `bin/cron/build_contact_rollups_v2`, I will call `collect_and_process_contacts` instead of `build_and_sync`.